nautobot_ssot/jobs/examples.py

```diff
@@ -441,6 +557,13 @@ def lookup_object(self, model_name, unique_id):
                 return Site.objects.get(name=unique_id)
             except Site.DoesNotExist:
                 pass
+        elif model_name == "prefix":
+            try:
+                return Prefix.objects.get(
+                    prefix=unique_id.split("__")[0], tenant__slug=unique_id.split("__")[1] or None
```

Comment:
@glennmatthews , is there a better way to get multiple keys from the `unique_id`?

Reply:
Not really -- ultimately it's up to your job what the `unique_id` contains, after all -- but it wouldn't be a bad idea to add a small helper function to wrap the `unique_id.split("__")` call.